app/models/period.rb

```diff
@@ -57,6 +57,16 @@ def blood_pressure_control_range
     (three_months_ago..end_date)
   end
 
+  alias_method :bp_control_range, :blood_pressure_control_range
+
+  def bp_control_range_start_date
+    bp_control_range.begin.next_day.to_s(:day_mon_year)
+  end
+
+  def bp_control_range_end_date
+    bp_control_range.end.to_s(:day_mon_year)
+  end
```

Comment:
We use these begin / end dates enough that its worth exposing them right off the Period.